### PR TITLE
Add HTML for speakers page

### PR DESCRIPTION
**Pull request by Ctannerweb**
@acacheung :eyes: Changes to the content and slight layout changes could still happen after the homepage design is complete.

### CODE REVIEW

File: speakers.html [hunk 1]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+<head>
+  <meta charset="UTF-8">
+  <title>UX East: Camp</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="/stylesheets/screen.css">
+  <link rel="stylesheet" type="text/css" href="/stylesheets/nav.css">
```
Comment (acacheung) [inline speakers.html]: remove artifacts that you will not be using

File: speakers.html [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+  </script>
+  <script src="//use.typekit.net/kbm1iso.js"></script>
+  <script>try{Typekit.load();}catch(e){}</script>
+  <script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <script type="text/javascript" src="js/menu.js"></script>
+</head>
+<body>
+  <nav class="nav">
+    <div class="nav__bar"></div>
```
Comment (acacheung) [inline speakers.html]: ?

File: speakers.html [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+      </a>
+      <a class="nav__icon" href="#">
+        <span class="nav__icon__top"></span>
+        <span class="nav__icon__middle"></span>
+        <span class="nav__icon__bottom"></span>
+      </a>
+      <div class="nav__links">
+        <a class="nav__link" href="speakers.html">Speakers</a>
+        <a class="nav__link" href="">Attend</a>
+        <a class="nav__link" href="faq.html">faq</a>
```
Comment (acacheung) [inline speakers.html]: Order doesn't match psd and should be Faq

File: speakers.html [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+        <a class="nav__link" href="schedule.html">Schedule</a>
+      </div>
+    </div>
+  </nav>
+  <header class="header">
+    <h1 class="heading">Call for Proposals</h1>
+    <p class="heading__desc">We&lsquo;re currently accepting speaker proposals for UX East! We are looking for presentations anywhere between 30 - 60 minutes, and workshop proposals. Presentations will take place during the two main days of the conference, and will be followed by a Q&A session and discussion. Share your knowledge and insights of UX design in this unique setting.</p>
```
Comment (acacheung) [inline speakers.html]: smart quote is facing the wrong way

Comment (acacheung) [inline speakers.html]: ampersane should be &amp;

File: speakers.html [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+    <p class="heading__desc">Speakers will be notified by June 15, 2015.</p>
+    <a class="cta" href="">submit proposal</a>
```
Comment (acacheung) [inline speakers.html]: Submit proposal or Submit Proposal

File: speakers.html [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+      <h3>How is this different from a regular conference?</h3>
+      <p>UX Camp is a curated experience for a small group of motivated learners. Our attendees are open-minded and ready to collaborate, ask questions and share their process.</p>
+    </div>
+    <div class="faq__question">
+      <h3>Wait - only 30 attendees?</h3>
+      <p>We keep the camp small intentionally to allow for better connections between individuals. With the type of people who attend, you will get the same amount of interaction and questions - if not more - as you would get from a 100+ crowd in a traditional conference talk.</p>
+    </div>
+  </section>
+  <section class="speakers__prev">
+    <div class="prev__speaker">
```
Comment (acacheung) [inline speakers.html]: strange speakers__prev -> prev__speaker

File: speakers.html [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+        <img class="speaker__bio__image" src="images/speakers/ed_king.png" alt="Speaker Image" title="Ed king" />
+        <h3 class="speaker__bio__name">Ed king &amp; JD Jordan</h3>
+        <p class="speaker__bio__position">Director Of Strategy at <a href="">MaxMedia</a></p>
+      </div>
+      <div class="speaker__talk">
+        <h3 class="speaker__talk__title">It’s not mobile first, it’s user first</h3>
+        <p class="speaker__talk__desc">Ed and JD conducted a talk and workshop in one, exploring the emergence of mobile first design and how it can enrich UX. If we can solve our design problems under these constraints, we have conquered much of the mental overhead and can then enhance the experience for larger screen sizes.</p>
+      </div>
+    </div>
+    <a class="cta" href="">ok i&lsquo;m in!</a>
```
Comment (acacheung) [inline speakers.html]: Doesn't follow psd copy

File: speakers.html (hunk 2 above)
Comment (Ctannerweb) [inline speakers.html]: Its straight from the WGE website. I believe it is for the mobile nav.

File: speakers.html [hunk 8]
```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+<head>
```
Comment (acacheung) [inline speakers.html]: not indented properly

File: speakers.html (hunk 3 above)
Comment (Ctannerweb) [inline speakers.html]: I matched it to the latest homepage order. Will fix to Faq

File: speakers.html (hunk 2 above)
Comment (acacheung) [inline speakers.html]: correct! disregard

File: speakers.html [hunk 9]
```diff
@@ -0,0 +1,140 @@
...
+          <a class="sponsor__name" href="" target="_blank">+ Be this Bronze Sponsor</a>
+        </li>
+        <li class="sponsor tbd">
+          <svg class="sponsor__logo">
+            <use></use>
+          </svg>
+          <a class="sponsor__name" href="" target="_blank">+ Be this Bronze Sponsor</a>
+        </li>
+      </ul>
+      <p class="footer__dockyard">Made by <a href="http://www.dockyard.com">DockYard</a></p>
```
Comment (acacheung) [inline speakers.html]: always link to http://dockyard.com/

File: speakers.html [hunk 10]
```diff
@@ -0,0 +1,140 @@
...
+          <img class="speaker__bio__image" src="images/speakers/ed_king.png" alt="Speaker Image" title="Ed king" />
+          <h3 class="speaker__bio__name">Ed king &amp; JD Jordan</h3>
+          <p class="speaker__bio__position">Director Of Strategy at <a href="">MaxMedia</a></p>
+        </div>
+        <div class="speaker__talk">
+          <h3 class="speaker__talk__title">It’s not mobile first, it’s user first</h3>
+          <p class="speaker__talk__desc">Ed and JD conducted a talk and workshop in one, exploring the emergence of mobile first design and how it can enrich UX. If we can solve our design problems under these constraints, we have conquered much of the mental overhead and can then enhance the experience for larger screen sizes.</p>
+        </div>
+      </div>
+      <a class="cta" href="">ok, i&rsquo;m in!</a>
```
Comment (acacheung) [inline speakers.html]: we have a mix of ’ and `&rsquo;`. let's just stick to ’

Comment (acacheung) [inline speakers.html]: Capitalize Ok

File: speakers.html [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+          <a class="nav__link" href="speakers.html">Speakers</a>
+          <a class="nav__link" href="">Attend</a>
+          <a class="nav__link" href="faq.html">Faq</a>
+          <a class="nav__link" href="schedule.html">Schedule</a>
+        </div>
+      </div>
+    </nav>
+    <header class="header">
+      <h1 class="heading">Call for Proposals</h1>
+      <p class="heading__desc">We&lsquo;re currently accepting speaker proposals for UX East! We are looking for presentations anywhere between 30 - 60 minutes, and workshop proposals. Presentations will take place during the two main days of the conference, and will be followed by a Q&A session and discussion. Share your knowledge and insights of UX design in this unique setting.</p>
```
Comment (acacheung) [inline speakers.html]: we should do `&amp;` here instead of &
http://stackoverflow.com/questions/3493405/do-i-really-need-to-encode-as-amp

**Pull request merged**